src/fusion_power_plant.h

```diff
@@ -46,21 +47,30 @@ class fpp : public cyclus::Facility  {
 
   #pragma cyclus note {"doc": "A stub facility is provided as a skeleton " \
                               "for the design of new facility agents."}
-
-  /// A verbose printer for the fpp
+  //Functions:
+  /// A verbose printer for the FusionPowerPlant
   virtual std::string str();
 
-  /// The handleTick function specific to the fpp.
+  /// The handleTick function specific to the FusionPowerPlant.
   /// @param time the time of the tick
   virtual void Tick();
 
-  /// The handleTick function specific to the fpp.
+  /// The handleTick function specific to the FusionPowerPlant.
   /// @param time the time of the tock
   virtual void Tock();
 
+  //Member Variables:
+
+  //Resource Buffers:
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_storage;
+  cyclus::toolkit::ResBuf<cyclus::Material> tritium_excess;
+  cyclus::toolkit::ResBuf<cyclus::Material> helium_excess;
+  cyclus::toolkit::ResBuf<cyclus::Material> blanket_storage;
+  cyclus::toolkit::ResBuf<cyclus::Material> blanket_excess;
```

Comment:
Reposting the question about ResBuf vs Material for this updated file

Reply:
Resolved as of our conversation yesterday